functions/messages.js

```diff
@@ -202,11 +202,20 @@ const renderDeliveryDM = (ticketID, deliveryDMContent, deliveryChannel) => (
             type: 'button',
             text: {
               type: 'plain_text',
-              text: 'Email me a shopping list'
+              text: 'Email me this shopping list'
             },
             style: 'primary',
             action_id: 'email_shopping_list',
             value: ticketID
+          },
+          {
+            type: 'button',
+            text: {
+              type: 'plain_text',
+              text: 'One list for all my tickets'
+            },
+            style: 'primary',
```

Comment:
Should we choose one of these buttons to be `primary` and use a different styling for the other?

Reply:
SGTM

Reply:
![2020-08-07-212236_1920x1080_scrot](https://user-images.githubusercontent.com/46406/89699346-391c5780-d8f4-11ea-81fb-c040b42371b9.png)
How does that look?